README.md

```diff
@@ -1,371 +1,209 @@
+yourself) know that this message is more important than the average logged
+message.
+
+> **TYPOGRAPHICAL NOTE**: When we use `console.error()` in code snippets, we'll
+> preface the output statements with `ERROR:` to differentiate them from other
+> logged messages:
 
-When we use `console.error()` in code snippets, we'll preface the output statements with `ERROR:` to differentiate them from other logged messages:
 ```js
 console.error('Uh oh, you done goofed.');
 // ERROR: Uh oh, you done goofed.
 ```
 
-You probably won't use `console.error()` very often — `console.log()` is by far the best tool for simple debugging — but now at least you'll know what's happening when you open up the JS console on a random site and see a bunch of bright red warnings.
+## Demonstrate `console.warn()`
 
-### `console.warn()`
-Ditto for the `console` object's `warn()` method, which provides an intermediate step between a regular `log()` message and a more dire `error()` message and is the source of those bright yellow messages you'll occasionally see in the browser's JS console:
+A step down in severity from `console.error()` is `console.warn()`. It provides
+step between a regular `log()` message and a more dire `error()` message.
 
 ![`console.warn()`](https://curriculum-content.s3.amazonaws.com/web-development/js/basics/intro-to-debugging-readme/console_error_log_warn.png)
 
-The `warn()` method is often used to alert developers looking at the JS console that they've done something inadvisable but not reaching the level of a full-blown error. For example, we could highlight the usage of a soon-to-be-deprecated feature:
-```js
-console.warn("Hmmm, you probably don't want to do that...");
-// WARN: Hmmm, you probably don't want to do that...
-```
+## Demonstrate `console.table()`
 
-When we use `console.warn()` in code snippets, we'll preface the output statements with `WARN:` to differentiate them from other logged messages.
+A very handy method to help work with `Array`s is `console.table()`:
+
+It prints a table of entries
+
+![Google Chrome documentation for console.table](https://developers.google.com/web/tools/chrome-devtools/console/images/table-people-objects.png)
 
 ## Conclusion
-Over the course of your programming career, you'll probably spend **significantly** more time debugging than actually writing new code. Just as your coding skills will improve with practice, so too will your debugging skills.
 
-Debugging can sometimes feel very demoralizing. You'll fix one bug and ten new ones appear:
+Over the course of your programming career, you'll probably spend
+**significantly** more time debugging than actually writing new code. Just as
+your coding skills will improve with practice, so too will your debugging
+skills. In fact, the [Google Chrome Console Docs][chromedocs] show _additional_
+methods on `console` that can format and display your code.
+
+
+Debugging can sometimes feel very demoralizing. You'll fix one bug and ten new
```

Comment:
I changed the demoralizing word to something else just because it seemed a bit bleak at this point in their career. I can definitely see leaving it in, but as Kellye called it out for me, I wanted to call it out for you.

Reply:
OK